src/main/java/supersymmetry/api/recipes/SuSyRecipeMaps.java

```diff
@@ -193,7 +193,7 @@ public class SuSyRecipeMaps {
             .setProgressBar(GuiTextures.PROGRESS_BAR_CIRCUIT, ProgressWidget.MoveType.HORIZONTAL)
             .setSound(GTSoundEvents.ASSEMBLER);
 
-    public static final RecipeMap<SimpleRecipeBuilder> RAILROAD_ENGINEERING_STATION_RECIPES = new RecipeMap<>("railroad_engineering_station", 16, 1, 4, 0, new SimpleRecipeBuilder(), false)
+    public static final RecipeMap<SimpleRecipeBuilder> RAILROAD_ENGINEERING_STATION_RECIPES = new RecipeMap<>("railroad_engineering_station", 12, 1, 3, 0, new SimpleRecipeBuilder(), false)
```

Comment:
why

Reply:
I concur here, especially given how some assembly line recipes do need all those slots. This change should be reverted.